src/containers/ProjectDashboardContainer.jsx

```diff
@@ -20,18 +22,57 @@ const propTypes = {
 };
 
 class ProjectDashboardContainer extends Component {
-
+  constructor() {
+    super();
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+    this.state = {
+      searchText: '',
+      translationLanguage: '',
+    };
+  }
   componentDidMount() {
     const { actions } = this.props;
     actions.fetchProject(this.props.params.project_id);
   }
 
+  handleSearch(event) {
+    this.setState({
+      searchText: event.target.value,
+    });
+  }
```

Comment:
Will this dispatch an action to load a translation for the selected language? If so, it'll need the language code.